docs/modules/ROOT/pages/getting-started/installation.adoc

```diff
@@ -13,7 +13,7 @@ This document explains how to install and configure KubeArchive in your Kubernet
 
 * A PostgreSQL instance protected with TLS (can be self-signed, KubeArchive does not verify it)
 * CertManager is installed on the Kubernetes cluster (+v1.9.1)
-* Knative Eventing is installed on the Kubernetes cluster (+v1.15.0)
+* Knative Eventing is installed on the Kubernetes cluster (+v1.17.4)
```

Comment:
I'm not sure about having these versions stated as "minimum required versions".

For example, in PostgreSQL we are testing everything with one version, but that doesn't imply that KubeArchive doesn't work with an older version.

I think it would be better to rephrase it as "tested with vx.y.z" instead of showing it as a requiredment.

Reply:
It is the minimum required version. Typically releases move forward, and it's much more likely to work on a later version than a previous, especially a later minor version.  I think it's pretty standard to phrase it the was it is here.

And if someone comes to us and says it doesn't work on an earlier version, are we going to fix it? No, we're going to tell them to upgrade. But if it doesn't work on a later version, we will fix it (at least we'll have to at some point).

Reply:
I think that having "tested with" leads to the same questions than having "required". If someone wants to run with an earlier version they would ask "you test X version, but does it work with X-1 version?" and I think the same goes with "required": "you require X version, but does it work with X-1 version?".  So given that, I think we should with a more strong statement like "required" instead of "tested". If we run into trouble because of this we can revisit.

Reply:
If we are going that path, can you please add the required version also for PostgreSQL?
Also, given the "requirement" part of the statement, I would suggest including a release note for that because it's going to potentially impact users.

Reply:
I added the version for PostgreSQL and the release note.